webrtc.html

```diff
@@ -2890,6 +2890,89 @@
             </pre>
           </dd>
 
+          <dt>Promise&lt;void&gt; replaceTrack (MediaStreamTrack withTrack)</dt>
+          <dd>
+            <p>Attempts to replace the track being sent with another track
+            provided, without renegotiation.</p>
+
+            <p>When the <dfn id="dom-rtpsender-replacetrack">
+            <code>replaceTrack()</code></dfn> method is invoked, the user agent
+            MUST run the following steps:</p>
+
+            <ol>
+              <li>
+                <p>Let <var>p</var> be a new promise.</p>
+              </li>
+
+              <li>
+                <p>Let <var>withTrack</var> be the argument to this method.</p>
+              </li>
+
+              <li>
+                <p>If <var>withTrack's</var> <code><a href=
+                  "getusermedia.html#dom-mediastreamtrack-kind">kind</a></code>
+                attribute differs from the </var><code><a href=
+                  "getusermedia.html#dom-mediastreamtrack-kind">kind</a></code>
+                of this sender's
+                <code><a href="#dom-rtpsender-track">track</a></code>,
+                then reject
+                <var>p</var> with <code>IncompatibleMediaStreamTrackError</code>,
+                return <var>p</var> and abort these steps.</p>
+              </li>
+
+              <li>
+                <p>Run the following steps in parallel:</p>
+
+                <ol>
+                  <li>
+                    <p>Determine if negotiation is needed to transmit
+                    <var>withTrack</var> in place of the sender's existing track.
+                    Ignore which <code>MediaStream</code> the track resides in and
+                    the <code>id</code> attribute of the track in this
+                    determination. If negotiation is needed, then reject
+                    <var>p</var> with <code>NegotiationNeededError</code>, return
```

Comment:
likewise, we should not mint a new error; probably should be using one of the options above (but different from the incompatible kind)